sites/partners/__tests__/components/listings/ListingFormActions.test.tsx

```diff
@@ -85,6 +85,10 @@ let partnerUser: User = {
   userRoles: { isPartner: true },
 }
 
+const doJurisdictionsHaveFeatureFlagOn = () => {
```

Comment:
You could add one test to this file to test the case where this returns true and the close button is not there